.github/workflows/covtest.yml

```diff
@@ -0,0 +1,60 @@
+name: Coverity Scan
+on:
+  schedule:
+  - cron: '0 0 * * 1,3,5'
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v1
+      - name: Install Qt
+        uses: jurplel/install-qt-action@v2
+        with:
+          version: 5.10.1
+          modules: qtcharts
+      - name: Coverity scripts
+        run: |
+          COVERITY_SCAN_PROJECT_NAME="mne-tools/mne-cpp"
+          COVERITY_SCAN_NOTIFICATION_EMAIL="christoph.dinh@mne-cpp.org"
```

Comment:
can we define multiple email addresses? If yes, please add your email and lorenzesch@hotmail.com. If not, please change to lorenzesch@hotmail.com.